scripts/runbiotools/gh2biotools.py

```diff
@@ -120,8 +121,9 @@ def print_summary(results):
 
 
 
-def run_upload(token, files):
-    headers = {**HEADERS, 'Authorization': f'Token {token}'}
+def run_upload(files):
+    TOKEN = os.environ['BIOTOOLS_API_TOKEN']
```

Comment:
minor point, this will exit with a KeyError exception if the environment variable is not defined.